ad-hoc-scripts/encyprt_nin_dob.sql

```diff
@@ -2,7 +2,40 @@ CREATE EXTENSION IF NOT EXISTS pgcrypto;
 
 --NOTE: YOU NEED TO EXCHANGE ::PUBLICKEY WITH BASE64 ENCODED PUBLIC KEY
 update cqc."Worker"
-set "DateOfBirthEncryptedValue" = armor(pgp_pub_encrypt("DateOfBirthValue" :: text, dearmor(convert_from(decode('::PUBLICKEY','base64'),'UTF8'))));
+set "DateOfBirthEncryptedValue" = armor(pgp_pub_encrypt("DateOfBirthValue" :: text, dearmor(convert_from(decode(
+'::PUBLICKEY'
+,'base64'),'UTF8'))));
 
 update cqc."Worker"
-set "NationalInsuranceNumberEncryptedValue" = armor(pgp_pub_encrypt("NationalInsuranceNumberValue" :: text, dearmor(convert_from(decode('::PUBLIC KEY','base64'),'UTF8'))));
+set "NationalInsuranceNumberEncryptedValue" = armor(pgp_pub_encrypt("NationalInsuranceNumberValue" :: text, dearmor(convert_from(decode(
+'::PUBLIC KEY'
+,'base64'),'UTF8'))));
+
+-- OR BOTH TOGETHER
```

Comment:
Is it worth choosing which approach to go for so the script itself can be run?

Also typo in filename 😄

Reply:
Doing them together is probably cleaner but either or 🤷

Reply:
I just had to google where the typo is 🙈 ah dyslexia.       Running them together is faster but I have had to run them separately as well (when debugging etc) so probably best to keep both in, but Il write a note that running them together is better when deploying.

Reply:
Ah! Sorry bud, I should've pointed it out.

OK, cool. Sounds good 👍